### PR TITLE
build libicu_io when -sUSE_ICU=1

### PR DESCRIPTION
**Pull request by PikachuHy**
fix https://github.com/emscripten-core/emscripten/issues/20180

### CODE REVIEW

File: tools/ports/icu.py [hunk 1]
```diff
@@ -78,14 +78,16 @@ def create_libicu_i18n(lib_output):
   return [
       shared.cache.get_lib(get_lib_name(libname_libicu_common, settings), create_libicu_common), # this also prepares the build
       shared.cache.get_lib(get_lib_name(libname_libicu_stubdata, settings), create_libicu_stubdata),
-      shared.cache.get_lib(get_lib_name(libname_libicu_i18n, settings), create_libicu_i18n)
+      shared.cache.get_lib(get_lib_name(libname_libicu_i18n, settings), create_libicu_i18n),
+      shared.cache.get_lib(get_lib_name(libname_libicu_io, settings), create_libicu_io)
```
Comment (kripken) [inline tools/ports/icu.py:82]: It doesn't look like `create_libicu_io` is defined in this PR?

Comment (PikachuHy) [reply tools/ports/icu.py:90]: sorry
fixed

**Comment by sbc100** [comment]
Can you fix the flake8 warning: `./tools/ports/icu.py:19:1: E302 expected 2 blank lines, found 1`

**Comment by sbc100** [comment]
The chrome failures are known/unrelated failures.  Landing without waiting for tests to pass.

**Pull request merged**